.travis.yml

```diff
@@ -1,2 +1,3 @@
+sudo: false
 language: go
-go: 1.3
+go: 1.5
```

Comment:
Hmm, I'd rather keep the minimum as low as can be, this is because any breaking changes in 1.5 won't be noticed for majority of the users who don't use go1.5.